### ISSUE TITLE
Acceptance suite cleanup fails locally under Rails 5

### ISSUE THREAD

**Issue by derekprior** [thoughtbot/scenic#177]
The `acceptance_helper.rb` file speciefies that the dummy database should be dropped and recreated after the acceptance suite is run. Rails 5 added some protections to `rake db:drop` so that it will not drop a database that is not explicitly marked as the test database.

When I run my tests locally, this fails because the databases as created in `bin/setup` lack this information, I think.  

This works fine in travis, probably because RAILS_ENV is always equal to "test"? Not clear.

**Comment by teoljungberg** [comment]
Interestingly enough, I do not have this behaviour. My test suite runs fine under Rails 5.0, and the database is dropped afterwards.

Excerpt (disregard the warnings as they are resolved in an already proposed PR):

```sh
scenic master % BUNDLE_GEMFILE=/Users/teo/src/scenic/gemfiles/rails50.gemfile bundle exec rake

Randomized with seed 30769
.....................................................................

Finished in 0.2467 seconds (files took 0.88691 seconds to load)
69 examples, 0 failures

Randomized with seed 30769

DEPRECATION WARNING: Directly inheriting from ActiveRecord::Migration is deprecated. Please specify the Rails release the migration was written for:

  class CreateSearchResults < ActiveRecord::Migration[4.2] (called from load at /Users/teo/src/scenic/gemfiles/.bundle/vendor/bin/rake:22)
DEPRECATION WARNING: Directly inheriting from ActiveRecord::Migration is deprecated. Please specify the Rails release the migration was written for:

  class UpdateSearchResultsToVersion2 < ActiveRecord::Migration[4.2] (called from load at /Users/teo/src/scenic/gemfiles/.bundle/vendor/bin/rake:22)
DEPRECATION WARNING: Directly inheriting from ActiveRecord::Migration is deprecated. Please specify the Rails release the migration was written for:

  class UpdateSearchResultsToVersion2 < ActiveRecord::Migration[4.2] (called from load at /Users/teo/src/scenic/gemfiles/.bundle/vendor/bin/rake:22)
DEPRECATION WARNING: Directly inheriting from ActiveRecord::Migration is deprecated. Please specify the Rails release the migration was written for:

  class CreateSearchResults < ActiveRecord::Migration[4.2] (called from load at /Users/teo/src/scenic/gemfiles/.bundle/vendor/bin/rake:22)
.DEPRECATION WARNING: Directly inheriting from ActiveRecord::Migration is deprecated. Please specify the Rails release the migration was written for:

  class CreateSearchResults < ActiveRecord::Migration[4.2] (called from load at /Users/teo/src/scenic/gemfiles/.bundle/vendor/bin/rake:22)
DEPRECATION WARNING: Directly inheriting from ActiveRecord::Migration is deprecated. Please specify the Rails release the migration was written for:

  class CreateChildren < ActiveRecord::Migration[4.2] (called from load at /Users/teo/src/scenic/gemfiles/.bundle/vendor/bin/rake:22)
DEPRECATION WARNING: Directly inheriting from ActiveRecord::Migration is deprecated. Please specify the Rails release the migration was written for:

  class UpdateChildrenToVersion2 < ActiveRecord::Migration[4.2] (called from load at /Users/teo/src/scenic/gemfiles/.bundle/vendor/bin/rake:22)
DEPRECATION WARNING: Directly inheriting from ActiveRecord::Migration is deprecated. Please specify the Rails release the migration was written for:

  class UpdateChildrenToVersion2 < ActiveRecord::Migration[4.2] (called from load at /Users/teo/src/scenic/gemfiles/.bundle/vendor/bin/rake:22)
DEPRECATION WARNING: Directly inheriting from ActiveRecord::Migration is deprecated. Please specify the Rails release the migration was written for:

  class CreateChildren < ActiveRecord::Migration[4.2] (called from load at /Users/teo/src/scenic/gemfiles/.bundle/vendor/bin/rake:22)
..Dropped database 'dummy_test'
Created database 'dummy_test'


Finished in 30.81 seconds (files took 0.07528 seconds to load)
3 examples, 0 failures
```

**Comment by derekprior** [comment]
Those deprecations are a separate issue from the recently merged change, which fixed the ones generated by the spec suite. See #176.

**Comment by teoljungberg** [comment]
@derekprior That's why I denoted it and opened #179 to remedy this.

**Comment by teoljungberg** [comment]
@derekprior Since I'm not able to reproduce this issue on my machine(which was setup for the first time with Scenic and Rails 5). Do you think there's something going on with your setup?

**Comment by devonestes** [comment]
Are you getting this error?

```
ActiveRecord::NoEnvironmentInSchemaError: 

Environment data not found in the schema. To resolve this issue, run: 

    bin/rails db:environment:set RAILS_ENV=test
```

I frequently run into issues dropping databases because of that added security issue in Rails 5. I've seen it fixed by running that rake task described in the error message. If you really want to ignore those warnings, you can set an environment variable to disable them `DISABLE_DATABASE_ENVIRONMENT_CHECK=1`.

**Comment by derekprior** [comment]
Yes. That's the error I get. To be able to close this issue, I want to:

1. Delete any dummy databases already on my machine (`dummy_test`,
`dummy_development`)
2. Get a fresh clone the scenic repo
3. Run bin/setup
4. Run rake
5. Run rake again (maybe - I think I remember this being triggered on the
second run).

If this means the `bin/setup` or `acceptance_helper.rb` files have to
change to run the right rake task or set an environment variable or
something like that, so be it.

**Comment by derekprior** [comment]
Fixed by #185
